static/js/settings_bots.js

```diff
@@ -27,6 +27,8 @@ exports.type_id_to_string = function (type_id) {
         return i18n.t("Incoming webhook");
     } else if (type_id === 3) {
         return i18n.t("Outgoing webhook");
+    } else if (type_id === 4) {
+        return i18n.t("Embedded bot");
```

Comment:
Agreed!
But I don't think the number would increase a lot hereafter, as almost all bot type cases are covered.

But definitely, using a mapping would be a plus. Will work on it.